araproc/framework/constants.py

```diff
@@ -15,3 +15,5 @@
 num_samples_per_dda = 32768
 num_polarization = 2
 num_trigger_type = 3
+num_RF_channels = 16
```

Comment:
Given how closely tied this is to `rf_channels_ids`, probably its better to define as `num_RF_channels = len(rf_channels_ids)`

Reply:
Good idea!